### PR TITLE
hacktoberfest2022

### PR DESCRIPTION
**Pull request by varunsharma6956**

### CODE REVIEW

File: Recursions/Palindrome-check_recursion.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			return true;
+
+		return ((str.charAt(start)==str.charAt(end)) && 
+			     isPalindrome(str, start + 1, end - 1));
+	}
+    public static void main(String [] args) 
+    {
+    	
+    	String s = "MelonoeeG";
+
```
Comment (sanatbali) [inline Recursions/Palindrome-check_recursion.java]: Change this MelonoeeG
Put something meaningfull

**Review by sanatbali — CHANGES_REQUESTED** [review]

**Comment by varunsharma6956** [comment]
Hey sir now please approve my request as changed the complete code and did
meaningful and approve it and merge it

**Review by sanatbali — CHANGES_REQUESTED** [review]
Please done string palindrome check and use meaningfull name!

**Comment by varunsharma6956** [comment]
Now done code updated and changed name Plaindrome-Recursion.java please now
accept it and merge

**Review by pranitrathod — APPROVED** [review]

**Pull request merged**